t04_refactored.py

```diff
@@ -218,9 +220,45 @@ def team_14_adv(username, delay):
 
 
 def team_15_adv(username, delay):
-    pass
-    # TODO Add your code here
-    return True
+
+    """
+    # https://docs.google.com/document/d/1Xi3PVM3_64dIIpbPdPyi13yFwu7lCRqUi7PjU0Fy0hI/edit?usp=sharing
+    #Tobore Takpor
+    #Jack Ngog
+    """
+
+
+
+
+
+    opponent = input("Who will you choose for your opponent? (Scott/Tojo/Bystander)")
+
+    isDead = False
+    if opponent == "Scott":
+        #bad choice
+        print('''As he steps into the ring he make nervous eye contact with you. He says "I was hoping no one would choose me, I didn't want it to come to this"''')
+        sleep(delay * 3)
+        print("Within a millisecond he teleports behind you wrapped around your waist and suplexes you, snapping your neck on the hard cold unforgiving concrete floor")
+        print("Dr.Scott has killed you")
+
+        return False
+
+    elif opponent == "Tojo":
+        #good choice
+        print("You get in the ring with Tojo. You notice he's moving a little slow and creaky.")
+        print("Turns out he overdid it on the bench press before the fight. You easily throw him to the ground.")
+        print("Unable to properly use his arms, he can't get up and is tapped out.")
+        sleep(delay)
+
+    else:
+        #nutral choes
+        print("The fellow student you fight happens to be on the exact same level as you so you tie.")
+        print("You have to try fighting them again  at the next BWT")
+        sleep(delay)
+        return True
```

Comment:
Move this left one indentation; you need to `return True` always (not just at the end of the `else` statement.